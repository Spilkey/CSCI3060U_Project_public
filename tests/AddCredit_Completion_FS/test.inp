loginuser02addcredit100logout