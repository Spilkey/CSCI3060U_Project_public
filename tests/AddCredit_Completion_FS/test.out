06 user02          FS 000100.00
00